Tests/Pcap++Test/Tests/IpMacTests.cpp

```diff
@@ -85,9 +84,23 @@ PTF_TEST_CASE(TestIPAddress)
 		PTF_ASSERT_FALSE(ip6Addr2.matchSubnet(subnetIp6Addr01, i));
 		PTF_ASSERT_FALSE(ip6Addr2.matchSubnet(subnetIp6Addr02, i));
 	}
-} // TestIPAddress
 
+	/* Test less-than comparison operator */
+	pcpp::IPv4Address IpV4_1("1.1.1.1");
+	pcpp::IPv4Address IpV4_2("1.1.1.1");
+	pcpp::IPv4Address IpV4_3("1.1.1.2");
+
+	PTF_ASSERT_FALSE(IpV4_1 < IpV4_2);
+	PTF_ASSERT_TRUE(IpV4_1 < IpV4_3);
+	PTF_ASSERT_FALSE(IpV4_3 < IpV4_1);
 
+	pcpp::IPv6Address ipv6Address("2001:db8::2:1");
+	pcpp::IPv6Address ipv6AddressLong("2001:db8:0:0:0:0:2:1");
+	pcpp::IPv6Address ipv6Address2("2001:db8::2:2");
+
+	PTF_ASSERT_FALSE(ipv6Address < ipv6AddressLong);
+	PTF_ASSERT_TRUE(ipv6Address < ipv6Address2);
```

Comment:
Few tests to add could be comparing objects of type `IPAddress`: IPv4 to IPv4, IPv6 to IPv6 and IPv4 to IPv6. Please let me know what you think.

Reply:
I have added further unittests